debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/util/TestHelper.java

```diff
@@ -314,7 +315,27 @@ protected static void disableDbCdc(SqlServerConnection connection, String name)
     }
 
     /**
-     * Enables CDC for a table if not already enabled and generates the wrapper
+     * Enables CDC for given schema and table if not already enabled and generates the wrapper
+     * functions for that table.
+     *
+     * @param connection
+     *            sql connection
+     * @param tableName
+     *            the name of the schema, may not be {@code null}
+     * @param tableName
+     *            the name of the table, may not be {@code null}
+     * @throws SQLException if anything unexpected fails
+     */
+    public static void enableSchemaTableCdc(SqlServerConnection connection, String schemaName, String tableName) throws SQLException {
```

Comment:
Would be nicer to overload `enableTableCdc()`, but unfortunately `enableTableCdc(SqlServerConnection, String, String)` already exists for enabling CDC for a table with a custom capture name.

Reply:
We may use `TableId` alternatively? It's the safer way for identifying tables to begin with.

Reply:
Good, idea, thanks for the pointer!